### DISCUSSION TITLE
Advice on training a style

### DISCUSSION THREAD

**Discussion by Maxadox** [TheLastBen/fast-stable-diffusion#1798] [Q&A]
Hi everyone
I need to train a graphic style on a data set of 100 images of a a particular  artist.  Can anyone tell me a goos setting to start?  how many training steps for 100 instances images ? And how many regularisation images ( I don't intent to mix  styles, just wanna create new images with style accuracy ) How important are captions in this case?
thanks in advance

**Comment by TheLastBen** [comment]
pick the 10 best images from the dataset, rename them to one unique unknown word, and train with them with the default settings, no captions, no regularization

**Comment by Maxadox** [comment]
thanks for the reply! Is there any other way to keep the 100 instances instead of 10? is it possible ?

**Reply by TheLastBen** [reply]
You can increase to 20 but more than that it could reduce the output quality
